.github/actions/analyze-test-runs/action.yml

```diff
@@ -39,3 +41,24 @@ runs:
       fi
       echo "No test execution found in build output, thus nothing to check."
       exit 0
+  - name: Unfinished tests
+    if: always()
```

Comment:
:question: Does it needs to be always? Only if test timedout or cancelled, right? But no harm in executing it always though.